src/custom/pages/Profile/index.tsx

```diff
@@ -208,10 +208,16 @@ export default function Profile() {
             </ConvertWrapper>
 
             <CardActions>
-              <ExtLink href={getBlockExplorerUrl(chainId || 1, V_COW_CONTRACT_ADDRESS[chainId || 1], 'address')}>
+              <ExtLink
+                href={getBlockExplorerUrl(
+                  chainId || ChainId.MAINNET,
```

Comment:
seeing as you check chainId repeatedly here, you can also just set a default value to the chainId return from `useActiveWeb3React` like

`const { chainId = SupportedChainId.MAINNET, ... } = useActiveWeb3React()` 

just make sure that you're not depending on chainId being undefined somewhere. As in, we block sensitive stuff if there's no connection

Reply:
Updated. As for the `undefined` depedency I see these:

> const cow = useTokenBalance(account || undefined, chainId ? COW[chainId] : undefined)

>       {chainId && chainId !== ChainId.MAINNET && (
        <NotificationBanner isVisible level="info" canClose={false}>
          Affiliate data is only available for Ethereum. Please change the network to see it.
        </NotificationBanner>
      )}

> {chainId && chainId === ChainId.MAINNET && <AffiliateStatusCheck />}

I think in case of the token balance, the `undefined` there becomes obsolete (as it will always get the mainnet balance) correct? Used here:

>   const cowBalance = formatSmartLocaleAware(cow, AMOUNT_PRECISION) || '0'
  const cowBalanceMax = formatMax(cow, COW_DECIMALS) || '0'
  
Should I simply remove the `undefined` in the `cow` const?

Reply:
i think it's fine as is, no worries 👌